client/js/analysisui/layoutcell.js

```diff
@@ -65,9 +72,9 @@ var LayoutCell = Backbone.View.extend({
 
         if (this.$previousContent !== null)
             this.trigger('layoutcell.contentChanged', Math.random());
```

Comment:
might be worth providing a comment describing the rationale for `Math.random()` here... seems a bit... random.